build-tools/helm/spark-kubernetes-operator/values.yaml

```diff
@@ -14,9 +14,9 @@
 # limitations under the License.
 
 image:
-  repository: spark-kubernetes-operator
+  repository: apache/spark-kubernetes-operator
```

Comment:
BTW, if you follow the full instructions, the following image exists in your local docker system like the following.
```
$ docker images
REPOSITORY                  TAG               IMAGE ID       CREATED         SIZE
spark-kubernetes-operator   0.1.0-SNAPSHOT    05633213ce49   5 seconds ago   573MB
```